src/views/rss.nimf

```diff
@@ -54,6 +50,12 @@ Twitter feed for: ${desc}. Generated by ${cfg.hostname}
 <img src="${urlPrefix}${getPicUrl(card.image)}" style="max-width:250px;" />
 #  end if
 #end if
+#if tweet.quote.isSome and get(tweet.quote).available:
+#  let quoteLink = getLink(get(tweet.quote))
+<hr/>
+<p>Quoting: <a href="${urlPrefix}${quoteLink}">${cfg.hostname}${quoteLink}</a></p>
+${renderRssTweet(get(tweet.quote), cfg)}
+#end if
```

Comment:
Let me start by saying that this is the sole reason it's keeping me away from using Nitter's RSS feeds and I'm grateful you pointed in the right direction here. I hope this get's merged as soon as possible 🤞 

My 2 cents here would be that this should use more semantically correct element: 
```suggestion
<blockquote>
<p>
$renderRssTweet(get(tweet.quote), cfg)}
</p>
<footer>
— <cite><a href="${urlPrefix}${quoteLink}">${cfg.hostname}${quoteLink}</a></cite>
</footer>
</blockquote>
```

Reply:
Thanks for the suggestion! It looks much better than what I did. Sorry I didn't merge it before, I was busy and then Twitter started breaking Nitter. Let's see if things will get more stable soon and I'll go back to this!